contents/teams/replay/objectives.mdx

```diff
@@ -1,17 +1,19 @@
-### 🎯 Goal 1: 10 Happy Android Replay Users 
-
-We'll also release an alpha of iOS Mobile Replay.
+### 🎯 Goal 1: Quality and Consolidation
 
 Why? 
 
-We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
+We've done a lot of investigation and preparation work and we want to captilise on that
+
+* Mobile replay out of beta for Android/iOS
+* Build the best UX for capturing and playing recordings
+* Build the best UX for finding recordings
 
-### 🎯 Goal 2: People love replay even more
+### 🎯 Goal 2: Error tracking MVP
 
-Why? - we want this product to be fit for everyone and understand where it doesn't spark joy and how we can increase the value people receive and feel.
+Why? PostHog is "The single platform to analyze, test, observe, and deploy new features". Having network timings, network payloads, and console errors in Session Replay is a great developer tool. We want to learn by doing to understand how we could capture and expose even more information here to help people dig in to what is happening with their products
```

Comment:
I'd define a few tasks here since even error tracking as MVP is a huge milestone already.

A few things that come to my mind.

* SDK work (capture API, error call back hook up - this is per framework, source maps linking, SDK bundle size).
* Symbols uploading and linking (source maps for now) - need a CLI
* Ingestion (probably reusing `$exception`)?
* Symbolication
* Grouping - signal vs noise
* Frontend Issues list - read, resolve, etc
* Alerting

We can define what our MVP is gonna do and list them?

Reply:
yeah probably not needed, you're right, just a braindump